### DISCUSSION TITLE
Group by

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3758] [Q&A]
Hi,

I wanted to find sum of column A in a table grouping by the column B. Is it possible to do currently?

Thanks and Regards,
Sumathy Venugopal

**Comment by dstala** [comment]
@MGW-SumathyV 

This helps?
https://docs.nocodb.com/setup-and-usages/rollup#rollup

**Reply by MGW-SumathyV** [reply]
I tried using rollup but for my use case rollup requires some manual task hence groupby feature would make it faster
Do you have any idea when the feature will be released
